portfolio/src/main/webapp/index.html

```diff
@@ -30,12 +30,13 @@ <h2 id="project-name"></h2>
           <p id="project-text"></p>
           <img id="project-img" src="" width="100%"></img>
       </div>
-      <button onclick="displayMessages()">
-        Messages
-      </button>
-      <div id="messages-container">
-          <ul id="messages-list">
+      <div id="comments-container">
+          <ul id="comments-list">
           </ul>
+          <form action="/addcomment" method="POST">
```

Comment:
can we add the form to be at the top of the existing comments? otherwise if you become super famous and get 1k+ comments, one would have to scroll all the way down to leave a comment :)